docs/api-ref.md

```diff
@@ -4042,4 +4055,49 @@ server.workbooks.update_conn(workbook, connection)
 <br>
 <br> 
 
+#### workbooks.populate_pdf
+```
+workbooks.populate_pdf(workbook_item, req_options=None)
+```
+
+Populates the PDF content of the specified workbook.
+
+This method uses the `id` field to query the PDF content, and populates the content as the `pdf` field.
```

Comment:
_Not sure I understand this, but taking a crack based on my understanding of the REST endpoint. Also suggest this change for line 3354 in this review._

Suggest:

This method populates a PDF with image(s) of the workbook view(s) you specify.